src/x86/up.c

```diff
@@ -155,7 +156,7 @@ mraa_up_board()
     mraa_up_set_pininfo(b, 8, "UART1_TX",  (mraa_pincapabilities_t){ 1, 1, 0, 0, 0, 0, 0, 1 }, 14);
     mraa_up_set_pininfo(b, 9, "GND",       (mraa_pincapabilities_t){ 0, 0, 0, 0, 0, 0, 0, 0 }, -1);
     mraa_up_set_pininfo(b, 10, "UART1_RX", (mraa_pincapabilities_t){ 1, 1, 0, 0, 0, 0, 0, 1 }, 15);
-    mraa_up_set_pininfo(b, 11, "GPIO17",   (mraa_pincapabilities_t){ 1, 1, 0, 0, 0, 0, 0, 0 }, 17);
+    mraa_up_set_pininfo(b, 11, "UART1_RTS",(mraa_pincapabilities_t){ 1, 1, 0, 0, 0, 0, 0, 1 }, 17);
```

Comment:
Hi the maximum name size is 12 character
include/mraa_internal_types.h:    char name[MRAA_PIN_NAME_SIZE]; /**< Pin's real world name */
api/mraa/common.h:#define MRAA_PIN_NAME_SIZE 12